### ISSUE TITLE
Accessibility Issues on Signup and Login Form

### ISSUE THREAD

**Issue by bbertucc** [bbertucc/equalify#341]
## Bugs
1. Signup and login page uses placeholder text to describe inputs. Placeholder text shouldn't be use for any inputs. Placeholder text fails color contrast checks and isn't read by screen readers once content is being added. If you did need to describe an input, use  `p` tag and associate it with the input using `aria-describedby`.
2. Signup and login form alerts are not read by screen readers. To fix, see inline descriptions on this link: https://www.w3.org/WAI/tutorials/forms/notifications/#inline
3. Signup page "First Name" labels are incorrectly attributed to inputs. The "First name" has the parameter `for="email"`. That should be set to `for="firstName"`. 
4. The form description is currently set as `<h2>`. It should be a `<p>` since it is not a section heading. 

## Additional Items
- **Please remove emojis from the form descriptions.** Emojis are read out weirdly on screen readers, so there's a general feeling in the a11y community that pages with emojis aren't cool. (This has been a hard fact for me because I love emojis.)
- **Update the descriptions.** "Begin your journey towards a more accessible world. Let's make accessibility a norm, together!" can become "Fill out this form to begin your accessibility journey with Equalify.". "Welcome back to Equalify, where accessibility is for everyone!" can become "Welcome back to Equalify! Please sign in:"

## Affected Pages
- https://staging.equalify.dev/login
- https://staging.equalify.dev/signup

## Additional Notes
@kevinandrews1 first reported issues 1 and 2 in this ticket: https://github.com/EqualifyEverything/equalify/issues/339

**Comment by wilsuriel03** [comment]
This has been resolved, coming up on the PR

**Comment by bbertucc** [comment]
@kevinandrews1 this is ready for your review

**Comment by kevinandrews1** [comment]
@bbertucc Ah, thanks, this is good to go and be closed, but I can't close the issue--probably cauz I didn't create it? NOt sure. Maybe check my GitHub perms in the org.

**Comment by bbertucc** [comment]
Try it now @kevinandrews1
